src/main/java/com/mentalhealthapi/mentalhealthapi/dto/Disorder.java

```diff
@@ -6,6 +6,6 @@
     private int id;
     private String name;
     private String description;
-    private String dsm5Page;
+    private String pageNumber;
```

Comment:
The variable name "dsm5Page" is a name that is difficult for an outside programmer to understand. Making the change for it to be more easily understood is more inline with the naming convention standards of Baeldung. https://www.baeldung.com/java-clean-code#2-naming-convention

Reply:
I disagree with having pageNumber as a variable name. Page number to what? The variable name should have a context and meaning There was another pull request that had a better idea - to have the full name of the book, instead of the acronym. Something like: diagnosticStatisticalManualPageNumber. I will consider your suggestion and this one.